forgerock-auth/src/main/java/org/forgerock/android/auth/devicebind/DeviceBindAuthenticators.kt

```diff
@@ -185,7 +197,7 @@ internal class BiometricAndDeviceCredential(private val biometricInterface: Biom
 /**
  * Settings for all the none authentication is configured
  */
-internal class None(private val keyAware: KeyAware): DeviceAuthenticator {
+internal class None(private val keyAware: KeyAware) : DeviceAuthenticator {
```

Comment:
internal class None(private val keyAware: KeyAware) : DeviceAuthenticator - Remove Space
 
like this

internal class None(private val keyAware: KeyAware): DeviceAuthenticator